iso15118/evcc/evcc_config.py

```diff
@@ -0,0 +1,135 @@
+import json
+import logging
+from typing import List, Optional
+
+from aiofile import async_open
+from pydantic import BaseModel, Field, validator
+
+from iso15118.shared.messages.enums import UINT_16_MAX, EnergyTransferModeEnum
+
+logger = logging.getLogger(__name__)
+
+
+class EVCCConfig(BaseModel):
+    supported_energy_services: List[str] = Field(
+        None, max_items=4, alias="supportedEnergyServices"
+    )
+    is_cert_install_needed: bool = Field(None, alias="isCertInstallNeeded")
+    # Indicates the security level (either TCP (unencrypted) or TLS (encrypted))
+    # the EVCC shall send in the SDP request
+    use_tls: Optional[bool] = Field(None, alias="useTls")
+    sdp_retry_cycles: Optional[int] = Field(None, alias="sdpRetryCycles")
+    max_contract_certs: Optional[int] = Field(None, alias="maxContractCerts")
+    enforce_tls: bool = Field(None, alias="enforceTls")
+    supported_protocols: Optional[List[str]] = Field(
+        None, max_items=4, alias="supportedProtocols"
```

Comment:
We could also potentially have -20 WPT and-20 ACDP. Maybe better to set this to 6.

Reply:
Done